cognite/client/_api/iam.py

```diff
@@ -187,17 +187,49 @@ def __init__(self, config: ClientConfig, api_version: str | None, cognite_client
         super().__init__(config, api_version, cognite_client)
         self._LIST_LIMIT = 100
 
-    def create(self, client_credentials: ClientCredentials | None = None) -> CreatedSession:
+    def __call__(self, client_credentials: ClientCredentials | dict | None = None) -> CDFSession:
```

Comment:
Let's not use `__call__` for this. All other APIClients use this for iterating over the resource, so we should be consistent.

Some naming suggestions: `session_ctx`, `create_ctx`, `create_and_revoke`.